CHANGELOG.md

```diff
@@ -9,6 +9,32 @@ stages](https://github.com/kubernetes/community/blob/master/contributors/devel/s
 
 ## Unreleased
 
+## 0.2.0 - 2019-12-18
+
+The Monitoring [CRDs] feature is graduated from Alpha to Beta.
+
+### Added
+
+- Add "Last Hearbeat Time" to [Octant] plugin to visualize the last time the Agent / Controller reported its status.
+- Add OVS version to Agent's monitoring CRD.
+- Add instructions to run [Octant] and the Antrea plugin either in a Pod or as a process. A Dockerfile and YAML manifest are included for deploying [Octant] as a Pod.
+
+### Changed
+
+- Use [libOpenflow] Go library to manage OVS flows instead of ovs-ofctl binary.
+- Minor changes to the OVS pipeline, in particular to fix issues when using the netdev OVS datapath type.
+- Officially-supported Go version is no longer 1.12 but 1.13.
+
+### Fixed
+
+- Allow the Node to reach all local Pods to support liveness probes even in the presence of Network Policies.
+- Network Policy fixes:
+   * support "except" field for "ipBlock" selectors
+   * fix support for [default policies]
+   * faster Network Policy enforcement by letting the CNI server notify the Agent's Network Policy controller when Pods are created
+   * fix race condition that sometimes caused Network Policy span not to be updated properly and the Network Policy not to be disseminated to Nodes properly
+   * ignore policy rules using named ports instead of allowing all traffic
+
```

Comment:
We should add issue #128

Reply:
Good point. I missed it since we never tagged the issue with the release.